src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,109 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import ProductListDetails from './ProductListDetails';
+import './ProductList.scss';
 
 const ProductList = () => {
-  return <div>ProductList</div>;
+  // const [List, setList] = useState([]);
+
+  // useEffect(() => {
+  //   fetch('http://localhost:3000/data/listData.json', {
+  //     method: 'GET',
+  //   })
+  //     .then(res => res.json())
+  //     .then(data => {
+  //       setList(data);
+  //     });
+  // }, []);
+
+  return (
+    <div>
+      <section className="subVisual">
+        <div className="img">
+          <img src="./images/vs1.jpg" alt="sub visual" />
```

Comment:
이미지 src 경로는 public 하위의 절대경로이기 때문에 `.` 없어도 됩니다!
```suggestion
          <img src="/images/vs1.jpg" alt="sub visual" />
```

Reply:
수정완료했습니다 :)